python/pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "reality_apis"
+version = "1.0.1"
+authors = [
+  { name="Bentley Systems", email="ariana.carnielli@bentley.com" },
```

Comment:
is the "email" property mandatory?
typically we never expose our devs emails as the authors; we just list it as the company

Reply:
from what I understand it's not mandatory in the toml (https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#authors-maintainers) so I can take it out and l leave only `name="Bentley` Systems"`.

We will need to register an e-mail to pypi afterwards if we publish the package.